base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
We should elaborate on that a bit in the docstring then, probably with a cross-reference to the full ccall docs which I hope explain this in more detail. Something like "This function should not usually be called directly, it gets called on the output from `cconvert`"